### PR TITLE
Dev/nieadni

### PR DESCRIPTION
**Pull request by Nieadni**

### CODE REVIEW

File: src/main/java/com/github/manasmods/tensura/registry/BlockRegistry.java [hunk 1]
```diff
@@ -24,7 +18,8 @@
 
 import java.util.Collection;
 
-class BlockRegistry {
+public class BlockRegistry {
```
Comment (Charismara) [inline src/main/java/com/github/manasmods/tensura/registry/BlockRegistry.java:21]: revert it

File: src/main/java/com/github/manasmods/tensura/registry/TensuraRegistry.java [hunk 2]
```diff
@@ -16,17 +16,14 @@
...
-import net.minecraftforge.registries.RegistryBuilder;
+import net.minecraftforge.registries.*;
 
 import java.util.function.Supplier;
 
 /** Class with all registries which are used in this Mod */
 public class TensuraRegistry {
     private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, Tensura.MOD_ID);
-    private static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, Tensura.MOD_ID);
+    public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, Tensura.MOD_ID);
```
Comment (Charismara) [inline src/main/java/com/github/manasmods/tensura/registry/TensuraRegistry.java:26]: make this private again

**Review by Charismara — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/manasmods/tensura/registry/TensuraRegistry.java (hunk 2 above)
Comment (Nieadni) [reply src/main/java/com/github/manasmods/tensura/registry/TensuraRegistry.java]: done

File: src/main/java/com/github/manasmods/tensura/registry/BlockRegistry.java (hunk 1 above)
Comment (Nieadni) [reply src/main/java/com/github/manasmods/tensura/registry/BlockRegistry.java]: done

**Review by Charismara — APPROVED** [review]

**Pull request merged**